### PR TITLE
update peering info from VPC to Vnet for Azure installs

### PR DESCRIPTION
**Pull request by gil-rosario**
Current peering section within Astro - Install on Azure contains VPC peering info which is relevant for AWS not Azure. I updated the section to include Azure Vnet peering prerequisites info.

### CODE REVIEW

File: astro/install-azure.md [hunk 1]
```diff
@@ -33,18 +33,26 @@ For more information about managing Azure subscriptions with the Azure CLI, see
...
-
-- A CIDR block (RFC 1918 IP Space) no smaller than a `/19` range. You must ensure it does not overlap with the AWS VPC(s) that you will be peering with later. The default CIDR range is `172.20.0.0/19`.
-- VPC Name / ID for peering with Astronomer.
-- The IP addresses of your DNS servers.
+### VNet peering prerequisites (Optional)
+
+If any Azure resources are on a private network, you can choose between two options:
+
+ 1. Create a VNet Peering connection between Astronomer's VNet and the VNets for your broader network. 
+ 2. [Virtual Network Integration](https://docs.microsoft.com/en-us/azure/api-management/virtual-network-concepts?tabs=stv2)
```
Comment (jwitz) [inline astro/install-azure.md:41]:
```suggestion
 - Create a VNet Peering connection between Astronomer's VNet and the VNets for your broader network. 
 - [Virtual Network Integration](https://docs.microsoft.com/en-us/azure/api-management/virtual-network-concepts?tabs=stv2).
```

File: astro/install-azure.md [hunk 2]
```diff
@@ -33,18 +33,26 @@ For more information about managing Azure subscriptions with the Azure CLI, see
...
-- Create a VPC Peering connection between the Astronomer VPC and your network VPCs.
-
-To implement the second option, you'll additionally need:
-
-- A CIDR block (RFC 1918 IP Space) no smaller than a `/19` range. You must ensure it does not overlap with the AWS VPC(s) that you will be peering with later. The default CIDR range is `172.20.0.0/19`.
-- VPC Name / ID for peering with Astronomer.
-- The IP addresses of your DNS servers.
+### VNet peering prerequisites (Optional)
+
+If any Azure resources are on a private network, you can choose between two options:
```
Comment (jwitz) [inline astro/install-azure.md:38]:
```suggestion
To give Astro access to any Azure resources on a private network, you can choose between two options:
```

File: astro/install-azure.md [hunk 3]
```diff
@@ -33,18 +33,26 @@ For more information about managing Azure subscriptions with the Azure CLI, see
...
+
+ 1. Create a VNet Peering connection between Astronomer's VNet and the VNets for your broader network. 
+ 2. [Virtual Network Integration](https://docs.microsoft.com/en-us/azure/api-management/virtual-network-concepts?tabs=stv2)
+
+ If you want to continue with the option #1, please open a ticket within our [Astronomer Support](https://support.astronomer.io/) with following details and support will proceed on configuring the VNet peering:
```
Comment (jwitz) [inline astro/install-azure.md:43]:
```suggestion
 To implement the first option, contact [Astronomer Support](https://support.astronomer.io/) and provide the following details:
```

File: astro/install-azure.md [hunk 4]
```diff
@@ -33,18 +33,26 @@ For more information about managing Azure subscriptions with the Azure CLI, see
...
+
+ - Azure TenantID of the target VNet
+ - Subscription ID of the target VNet
+ - Region of the target VNet
+ - VNet ID of the target VNet
+ - ResourceGroup of the target VNet
+ - A CIDR block (RFC 1918 IP Space) no smaller than a `/19` range. You must ensure it does not overlap with the Azure VNet(s) that you will be peering with later. The default CIDR range is `172.20.0.0/19`.
+ - ResourceID - The resource ID is similar to the following example: /subscriptions/<Subscription Id>/resourceGroups/myResourceGroupA/providers/Microsoft.Network/virtualNetworks/myVnetA (you can capture the resource ID within the Azure portal by following step #7 under [this link](https://docs.microsoft.com/en-us/azure/virtual-network/create-peering-different-subscriptions#portal). [Azure Portal login](https://portal.azure.com/)
```
Comment (jwitz) [inline astro/install-azure.md:51]:
```suggestion
 - ResourceID, for example: `/subscriptions/<Subscription Id>/resourceGroups/myResourceGroupA/providers/Microsoft.Network/virtualNetworks/myVnetA`. You can find the resource ID in the Azure portal by following step 7 in [Create peering - Azure portal](https://docs.microsoft.com/en-us/azure/virtual-network/create-peering-different-subscriptions#portal).
```

File: astro/install-azure.md [hunk 5]
```diff
@@ -33,18 +33,26 @@ For more information about managing Azure subscriptions with the Azure CLI, see
...
+ 
+ **Please Note:** As outline within the Azure docs [here](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-peering#permissions), the accounts you use to work with virtual network peering must be assigned to the following roles:
+        -  Network Contributor: For a virtual network deployed through Resource Manager.
+        -  Classic Network Contributor: For a virtual network deployed through the classic deployment model.
```
Comment (jwitz) [inline astro/install-azure.md:55]:
```suggestion
From here, Astronomer support will set up the VNet connection to Astro.
```
Is the account information specific to Astro in any way? If they already are working in VNet, I assume they have the correct permissions on their account.

File: astro/install-azure.md (hunk 1 above)
Comment (jwitz) [inline astro/install-azure.md:41]: What does the user need to do for [Virtual Network Integration](https://docs.microsoft.com/en-us/azure/api-management/virtual-network-concepts?tabs=stv2)?

**Review by jwitz — CHANGES_REQUESTED** [review]
Thank you @gil-rosario , looks great! Requesting changes on a small question I have towards the end of the section - wondering whether we need to specify account requirements for our own setup if they're completed as part of the initial VNet setup. Also curious about what the user needs to do for option 2 here.

**Comment by stuart23** [comment]
Hey @gil-rosario, you are pushing from a fork. Do you have access to create a new branch on this repo with your changes?

**Comment by jwitz** [comment]
Closing in favor of https://github.com/astronomer/docs/pull/1274

**Pull request closed without merging**